date_detection/debug.py

```diff
@@ -0,0 +1,55 @@
+"""
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+"""
+This module will help in calling the main.py locally and debugging
+if any error occurs.
+This module has a RequestClass that mimics the flask request class.
```

Comment:
What is "flask"?

Reply:
The hello_world function gets a <request (flask.Request): HTTP request object>  argument when it is called by the UI.

As it was difficult to create an actual flask request locally, we chose to create a dummy request using our own class.